ts/defs.bzl

```diff
@@ -152,6 +153,10 @@ def ts_project(
             See https://www.typescriptlang.org/docs/handbook/compiler-options.html#compiler-options
             Typically useful arguments for debugging are `--listFiles` and `--listEmittedFiles`.
 
+        isolated_declarations: Whether to enforce that declaration output (.d.ts file) can be produced for a
```

Comment:
Now we're mixing `types` and `declarations`, but this is aligning with the tsconfig so we should use `declarations`?

I'm still uncertain if we should have done that :/